modules/imaging_browser/php/queryengine.class.inc

```diff
@@ -75,6 +248,14 @@ class QueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                 new Cardinality(Cardinality::MANY),
                 $ScanType,
             );
+            $items[] = new QCDictionaryItem(
+                $ScanType . "_Selected",
+                "Selected file for $ScanType for session",
+                $scope,
+                new \LORIS\Data\Types\Enumeration("true", "false"),
```

Comment:
Is there a reason for this to be an enumeration of true/false instead of a BooleanType?